chat-plugins/mafia.js

```diff
@@ -2476,6 +2478,15 @@ const commands = {
 			this.sendReplyBox(`${showOrl ? `Original Rolelist: ` : `Rolelist: `}${roleString}`);
 		},
 
+		playerroles: function (target, room, user) {
```

Comment:
Why are we adding this? If its because of the town color on dark mode buttons issue we should tweak the color (I've been meaning to test a good color for this)